protos/head.proto

```diff
@@ -34,6 +34,16 @@ enum HeadField {
     ALL = 15;
 }
 
+enum NeckJoints {
+    ROLL = 0;
```

Comment:
Do we need to redefine yaw pitch roll? We have it in orbita2d already. Maybe it could be factorized?